{{cookiecutter.project_slug}}/Makefile

```diff
@@ -49,10 +49,13 @@ help:
 	@echo "\tsetup-dev-host - setup dev requirements"
 	@echo "\tsetup-pre-commit-hooks - setup pre-commit hooks"
 	@echo "\tsetup-release-tools: - setup tools needed for releasing package"
-	@echo "\tsetup-cicd-test-stage-github: - setup tools needed for CI/CD test stage (GitHub)"
-	@echo "\tsetup-cicd-release-stage-github: - setup tools needed for CI/CD release stage (GitHub)"
+{%- if cookiecutter.hosting == "GitLab" %}
 	@echo "\tsetup-cicd-test-stage-gitlab: - setup tools needed for CI/CD test stage (GitLab)"
 	@echo "\tsetup-cicd-release-stage-gitlab: - setup tools needed for CI/CD release stage (GitLab)"
+{%- else %}
+	@echo "\tsetup-cicd-test-stage-github: - setup tools needed for CI/CD test stage (GitHub)"
+	@echo "\tsetup-cicd-release-stage-github: - setup tools needed for CI/CD release stage (GitHub)"
+{%- endif %}
```

Comment:
`gitlab` and `github` qualifiers are not necessayr in make target name

Reply:
Done